src/fcmp_pp/fcmp_pp_rust/fcmp++.h

```diff
@@ -145,6 +140,47 @@ struct ObjectSlice
   uintptr_t len;
 };
 
+struct TreeRootUnsafe;
+
+struct PathUnsafe;
+
+struct HeliosBranchBlindUnsafe;
+struct SeleneBranchBlindUnsafe;
+
+struct BlindedOBlindUnsafe;
+struct BlindedIBlindUnsafe;
+struct BlindedIBlindBlindUnsafe;
+struct BlindedCBlindUnsafe;
+
+struct OutputBlindsUnsafe;
+
+struct FcmpPpProveInputUnsafe;
```

Comment:
Now that the SA/L specific stuff is removed, should we call this just `FcmpProveInputUnsafe`?

Reply:
Renamed to `FcmpPpProveMembershipInput...` to keep it tied to FCMP++